src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentPart.Edit.cshtml

```diff
@@ -7,13 +7,12 @@
     ContentPart contentPart = Model.ContentPart;
     ContentTypePartDefinition typePartDefinition = Model.ContentTypePartDefinition;
     var description = typePartDefinition.Description();
-    var namedPart = typePartDefinition.PartDefinition.IsReusable() && typePartDefinition.Name != typePartDefinition.PartDefinition.Name;
-    var name = typePartDefinition.PartDefinition.Name.HtmlClassify();
+    var isNamedPart = typePartDefinition.IsNamedPart();
 }
 
-@if (namedPart)
+@if (isNamedPart)
```

Comment:
maybe drop the `isNamedPart` variable and use `typePartDefinition.IsNamedPart()` directly